plugins/sbt/graphql-java-codegen-sbt-plugin/src/main/scala/io/github/dreamylost/graphql/codegen/GraphQLCodegenPlugin.scala

```diff
@@ -202,18 +202,20 @@ class GraphQLCodegenPlugin(configuration: Configuration, private[codegen] val co
       }, graphqlCodegen := {
         sLog.value.info(s"Generating files: ${BuildInfo.toString}")
         val mappingConfigSupplier = buildJsonSupplier(jsonConfigurationFile.value.orNull)
+        val language = mappingConfigSupplier.map(m => if (m.get.getGeneratedLanguage == null) GeneratedLanguage.JAVA
```

Comment:
Can you please do the same code I suggested in Gradle/Maven plugins? For me, it seems a bit cleaner. Let me know what you think.

Reply:
The function interface looks better .